packages/engine-server/src/markdown/remark/noteRefsV2.ts

```diff
@@ -306,39 +306,45 @@ export function convertNoteRefASTV2(
   const refLvl = MDUtilsV4.getNoteRefLvl(proc());
   let { dest, vault, config, shouldApplyPublishRules } =
     MDUtilsV4.getDendronData(proc);
-
   if (MDUtilsV5.isV5Active(proc)) {
     shouldApplyPublishRules = MDUtilsV5.shouldApplyPublishingRules(proc);
   }
-
   if (link.data.vaultName) {
     vault = VaultUtils.getVaultByNameOrThrow({
       vaults: engine.vaults,
       vname: link.data.vaultName,
     })!;
   }
-
   if (!vault) {
     return {
       error: new DendronError({ message: "no vault specified" }),
       data: [],
     };
   }
-  let { prettyRefs, wikiLinkOpts } = compilerOpts;
+  const { wikiLinkOpts } = compilerOpts;
+  const sitePrettyRefConfig = _.isUndefined(procOpts.config?.site?.usePrettyRefs)
```

Comment:
can we use [DConfig.getProp](https://github.com/dendronhq/dendron/blob/512b476e8ac986887f25bc9a21bf7b189ea19ce9/packages/engine-server/src/config.ts#L114-L114) here?

Reply:
done!